learningresources/search_indexes.py

```diff
@@ -0,0 +1,26 @@
+"""
+Index information for Haystack.
+
+http://django-haystack.readthedocs.org/en/latest/tutorial.html
+"""
+from haystack.indexes \
+    import SearchIndex, Indexable, CharField  # pragma: no cover
+
+from learningresources.models import LearningResource  # pragma: no cover
+
+
+class LearningResourceIndex(SearchIndex, Indexable):  # pragma: no cover
+    """
+    Index configuration for the LearningResource model.
+    """
+    # This is a config file for Haystack; nothing to test.
```

Comment:
nit: I don't think this comment is necessary. The statement here is also true for model classes as long as they don't have methods, but we don't explicitly call that out, and these classes typically get "integration" testing coverage when views or apis are tested.